docs/conf/scalar-loki-stack-custom-values.yaml

```diff
@@ -0,0 +1,50 @@
+promtail:
```

Comment:
This is a question.
It seems that the current configuration will collect logs of Scalar DL only.
Is there any plan to collect logs of Scalar DB in the future?

Reply:
Yes we do.
I thought this getting-started is for Scalar DL only so I didn't add it.
I added the configuration in https://github.com/scalar-labs/helm-charts/pull/87/commits/fb40dce6f3c4e250426b93ed8744a6157bdb1db2 for Scalar DB (using regular expression) as well according to our Helm Chart for Scalar DB.

Reply:
Thank you!
I was able to collect logs of Scalar DB using this configuration in my test environment.